lib/transcoder_web/live/upload_live.ex

```diff
@@ -0,0 +1,58 @@
+defmodule TranscoderWeb.UploadLive do
+  alias Transcoder.Model.Video
+  use TranscoderWeb, :live_view
```

Comment:
The convention is to have `use` appear before aliases.
Have you considered including credo? Then running `mix credo` would yield this as an exception. I'll ignore other styling thingies that credo would have caught going forward.
Also, I'm not seeing any formatting errors, but `mix format` works a similar way (maybe you're already using it, then - never mind).

Reply:
I'm already using `mix format`. I wondered whether it reorders imports as well but I see that it doesn't. However, running `mix credo` (even with `--strict` flag) doesn't warn about import ordering 🧐

Reply:
I think we might need to adjust `.credo.exs` and add needed checks there.

Reply:
yup, found it 🙂 https://hexdocs.pm/credo/Credo.Check.Readability.StrictModuleLayout.html